apps/end-tenancy/acceptance/features/tenant-details.js

```diff
@@ -202,6 +227,22 @@ Scenario('I see the correct title if I am checking a second tenant', function *(
   I.see(tenantDetailsPage.name.content.checkAnother);
 });
 
+Scenario('I see the correct title if I am requesting an nldp for a second tenant', function *(
```

Comment:
its the wording - if you are "requesting an nldp" you have selected the value "request" from the field "what"